packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -1525,6 +1525,25 @@ mixin WidgetInspectorService {
     return false;
   }
 
+  /// Notify attached tools to navigate to an object's source location.
+  void _sendInspectEvent(Object? object){
+    inspect(object);
+
+    final _Location? location = _getSelectedSummaryWidgetLocation(null);
+    if (location != null) {
+      postEvent(
+        'navigate',
+        <String, Object>{
+          'fileUri': location.file, // URI file path of the location.
+          'line': location.line, // 1-based line number.
+          'column': location.column, // 1-based column number.
+          'source': 'FlutterWidgetInspector',
```

Comment:
nit: I would use the naming convention used for flutter service extensions so more like 
'flutter.inspector'